integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DbUtils.java

```diff
@@ -249,15 +249,15 @@ public static synchronized void startOracleDB(String dbBaseImageName, int dbPort
                                 new V1ContainerPort()
                                 .containerPort(dbListenerPort)
                                 .name("tns")
-                                .protocol(V1ContainerPort.ProtocolEnum.TCP)
+                                .protocol("TCP")
                                 .hostPort(dbListenerPort)))
                             .resources(new V1ResourceRequirements()
                                 .limits(limits)
                                 .requests(requests))
                             .terminationMessagePath("/dev/termination-log")
-                            .terminationMessagePolicy(V1Container.TerminationMessagePolicyEnum.FILE)))
-                    .dnsPolicy(V1PodSpec.DnsPolicyEnum.CLUSTERFIRST)
-                    .restartPolicy(V1PodSpec.RestartPolicyEnum.ALWAYS)
+                            .terminationMessagePolicy("File")))
+                    .dnsPolicy("CluserFirst")
```

Comment:
"ClusterFirst" not "CluserFirst"

Reply:
Good eyes! I looked through all of the changes for misspellings and missed this one.